SYCL/ESIMD/regression/copyto_char_test.cpp

```diff
@@ -0,0 +1,116 @@
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+//==- copyto_char_test.cpp - Test for using copy_to to copy char buffers -==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+#include <iostream>
+#include <memory>
+
+using namespace sycl::ext::intel::esimd;
+
+template <typename DataT>
+using shared_allocator = sycl::usm_allocator<DataT, sycl::usm::alloc::shared>;
+template <typename DataT>
+using shared_vector = std::vector<DataT, shared_allocator<DataT>>;
+
+using DataT = char;
+
+int main(int, char **) {
+  sycl::queue queue();

```

Comment:
() would cause a warning during compilation.
```suggestion
  sycl::queue queue;
```

Reply:
Fixed